apps/prairielearn/src/lib/db-types.ts

```diff
@@ -20,7 +20,11 @@ export const AssessmentsFormatForQuestionSchema = z.array(
   }),
 );
 
-export const JsonCommentSchema = z.union([z.string(), z.array(z.any()), z.record(z.any())]);
+export const JsonCommentSchema = z.union([
```

Comment:
Should we use `z.json()` instead? https://zod.dev/api?id=json It seems less strict then this though.